python/src/ops.cpp

```diff
@@ -4,6 +4,7 @@
 #include <ostream>
 #include <variant>
 
+#include <pybind11/functional.h>
```

Comment:
Any reason you added that?

Reply:
It was suggested by Pybind to fix some casting error but it's not needed. It wasn't even then.